lua/telescope/utils.lua

```diff
@@ -147,6 +147,27 @@ utils.filter_symbols = function(results, opts, post_filter)
   end
 end
 
+utils.path_reverse = (function(filepath, file_sep, dir_open_sep, dir_close_sep)
+  local dirs = vim.split(filepath, Path.path.sep)
```

Comment:
> This wouldn't be compatible on Windows with how we currently preserve backslash path separators for Windows.

I checked the code and saw the use of `Path.path.sep`. I've replaced the hardcoded "/" in my previous PR. Would this solve your remark above? @jamestrew

Reply:
Thanks, I think this looks good :+1: